game.html

```diff
@@ -12,13 +12,29 @@
 </head>
 <body>
     <header>
-        <img src="img/favicon_package_v0.16 (4)/android-chrome-96x96.png" alt="logo">
-        <h1>Dopłyń na Hel z <i>tripcity</i></h1>
-        <button>start</button>
+        <a href="index.html">
+            <img id="logo" src="img/favicon_package_v0.16 (4)/android-chrome-96x96.png" alt="logo">
+        </a>
+        <h1>Dopłyń na Hel z <i><a href="index.html">tripcity</a></i></h1>
     </header>
     <main>
+        <div id="buttons">
+            <button><b>start</b></button>
+            <button>instrukcja</button>
+        </div>
+        <p id="score"></p>
         <canvas width="650px" height="650px"></canvas>
     </main>
+    <div class="onGame-modal">
+        <div class="onGame-modal__wrapper">
+            <h2 class="onGame-modal__score"></h2>
+            <h2 class="onGame-modal__text">Chcesz zagrać ponownie?</h2>
+            <div class="onGame-modal__btn-wrapper" >
+                <a href="game.html"><button class="onGame-modal__btn onGame-modal__btn--play-again" type="button">Tak!</button></a>
+                <a href="index.html"><button class="onGame-modal__btn onGame-modal__btn--toLandingPage" type="button">Nie</button></a>
```

Comment:
mam uwagę do class onGame modal.
wg mnie powinno to być camel case do końca
onGameModal__.     a nie onGame-modal__
jak chcesz to zmieniać to bardzo łatwo jest to zrobić. Klikasz na tekst który chcesz żeby był zmieniony i wciskasz F2 wyświetla się input bez opisu z tekstem który ma być zmieniony.  Wpisujesz nowy tekst i zmienia się wszędzie gdzie jest ten tekst.